packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -164,6 +167,15 @@ export interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderI
    */
   className?: string;
 
+  /**
+   * Whether to render shimmer
+   * @deprecated Use `ShimmeredDetailsList` instead: https://developer.microsoft.com/en-us/fabric#/components/detailslist/shimmer
+   */
+  shimmer?: boolean;
```

Comment:
Is this PR adding a new prop and immediately deprecating it?

Reply:
This looks like the result of an incorrect merge. It seems that `6.0` was merged into this branch as this property is only available there and has since been removed. See: https://github.com/OfficeDev/office-ui-fabric-react/blob/1e1f4252a6278e2d4368fd1f3c84faebd2e09091/packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts#L171

Reply:
shimmer?:  is not part of my change.  should be removed ?

Reply:
This is why it's appearing in the `api.md` file.